### PR TITLE
babel register cache: make sure the cache file's directory exists

### PR DESCRIPTION
**Pull request by jonathanong**
otherwise there will be an error on save. easier than doing `mkdirp()` in all my apps :)

### CODE REVIEW

File: packages/babel-register/package.json [hunk 1]
```diff
@@ -8,12 +8,13 @@
...
+    "babel-core": "^6.3.13",
+    "babel-runtime": "^5.0.0",
     "core-js": "^1.0.0",
     "home-or-tmp": "^1.0.0",
-    "path-exists": "^1.0.0",
     "lodash": "^3.10.0",
-    "source-map-support": "^0.2.10",
-    "babel-core": "^6.3.13",
-    "babel-runtime": "^5.0.0"
+    "mkdirp": "^0.5.1",
```
Comment (jonathanong) [inline packages/babel-register/package.json]: sorry i just did a `npm --save mkdirp`. let me know if u want me to reorder this for a better diff

**Comment by hzoo** [comment]
looks good

**Comment by amasad** [comment]
When would it not be created? It defaults to home or tmp, unless you set it (and if you set it, it seems like you should create the directory)

**Comment by jonathanong** [comment]
if i set it myself. i set it in the `node_modules` folder so it's cached with everything else. yes, i currently set it myself, but it's annoying, especially when you have multiple apps that need it.

**Pull request merged**